### ISSUE TITLE
Decorator proposal has been changed in Stage 2

### ISSUE THREAD

**Issue by trotyl** [compat-table/compat-table#1003]
The current test of class & property decorator is still based on the proposal in stage 1, and currently there is no implementation that fullfils the proposal in Stage 2, nor Babel or TypeScript.

It's not correct while list decorator in Stage 2 but test the proposal for decorator in Stage 1.

**Comment by kangax** [comment]
I guess someone needs to write a Babel transform...

/cc @hzoo @boopathi @shinew

**Comment by chicoxyzzy** [comment]
@kangax this needs Babylon parser support first and this work was started by @DrewML recently :) https://github.com/babel/babylon/pull/236

**Comment by DrewML** [comment]
Yep! Just waiting on some clarifications from Yehuda and some decisions about versioning/breaking changes before we finish it out and move on to the transform.

**Comment by ljharb** [comment]
In the meantime, I think it would be appropriate to mark *nobody* as supporting decorators, since nobody supports stage 2 semantics. We should probably also update the tests so that they will help once somebody does support it.

**Comment by hybrist** [comment]
@kangax Would you accept a PR to update the decorator tests to follow the current version of the proposal?

**Comment by ljharb** [comment]
@jkrems that'd be great, yes! please update the results as well :-)

**Comment by hybrist** [comment]
Done: https://github.com/kangax/compat-table/pull/1047

**Comment by afmenez** [comment]
Can't this one be closed now that the patch was merged?

**Comment by ljharb** [comment]
@afmenez #1047 isn't merged yet.
